admin/src/app/Components/create-admin/create-admin.component.ts

```diff
@@ -1,50 +1,61 @@
 import { Component } from '@angular/core';
 import { MatDialogRef } from '@angular/material/dialog';
-import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { MatSnackBar } from '@angular/material/snack-bar';
+import { AbstractControl, FormBuilder, FormGroup, Validators } from '@angular/forms';
 import { HttpErrorResponse } from '@angular/common/http';
 
 import { AccountService } from 'src/app/account.service';
+import { ConfirmValidParentMatcher, CustomValidators } from 'src/app/Core/customValidators';
 
 @Component({
   selector: 'app-create-admin',
   templateUrl: './create-admin.component.html',
   styleUrls: ['./create-admin.component.css']
 })
+
 export class CreateAdminComponent {
-  public registrationForm: FormGroup;
+  public createAdminForm: FormGroup;
+  public hidePassword: boolean = true;
   public serverErrorResponse: string = '';
+  public passwordValidator = new ConfirmValidParentMatcher('notSame');
 
   public get email(): AbstractControl | null {
-    return this.registrationForm.get('email');
+    return this.createAdminForm.get('email');
   }
 
   public get password(): AbstractControl | null {
-    return this.registrationForm.get('password');
+    return this.createAdminForm.get('password');
+  }
+
+  public get confirmPassword(): AbstractControl | null {
+    return this.createAdminForm.get('confirmPassword');
   }
 
   public constructor(
+    private readonly accountService: AccountService,
     private readonly matDialogRef: MatDialogRef<CreateAdminComponent>,
     private readonly formBuilder: FormBuilder,
-    private readonly snackBar: MatSnackBar,
-    private readonly accountService: AccountService
+    private readonly snackBar: MatSnackBar
   ) {
-      this.registrationForm = this.formBuilder.group({
+      this.createAdminForm = this.formBuilder.group({
         email: [
-          '',
+          '', 
           [
             Validators.required,
             Validators.email
           ]
         ],
-        password: ['', [Validators.required]]
-      });
+        password: ['', [Validators.required]],
```

Comment:
Since you have single validator, you can use different approach
```suggestion
        password: ['', Validators.required],
```